components/motor/dmc4000/dmc.go

```diff
@@ -65,17 +65,17 @@ type Motor struct {
 
 // Config adds DMC-specific config options.
 type Config struct {
-	DirectionFlip    bool    `json:"dir_flip,omitempty"`         // Flip the direction of the signal sent if there is a Dir pin
-	MaxRPM           float64 `json:"max_rpm,omitempty"`          // RPM
-	MaxAcceleration  float64 `json:"max_acceleration,omitempty"` // RPM per second
+	DirectionFlip    bool    `json:"dir_flip,omitempty"`                     // Flip the direction of the signal sent if there is a Dir pin
+	MaxRPM           float64 `json:"max_rpm,omitempty"`                      // TODO(RSDK-568): revs_per_min?
+	MaxAcceleration  float64 `json:"max_acceleration_rpm_per_sec,omitempty"` // TODO(RSDK-568): revs_per_min_per_sec?
```

Comment:
confirming, `max_rpm` should stay just `rpm` right? (as per @JessamyT)

Reply:
I actually think in the DMC one max_rpm makes sense because it does in fact set an upper limit on the rpm to which you can ask the motor to go. Meanwhile, max_rpm in other motors such as a non-encoded gpio motor is potentially misleading because it's just used in a weird math way for estimation, and assumes no load etc, but idk what anyone decided to call it.

Reply:
But maybe the comment should make this more clear, i.e. by telling people that max_rpm is used to set an upper bound. And then if it's not changed in gpio motor, that should also have a comment saying it's used differently there.

Reply:
thanks for the context - yeah, I'm not sure of a better name for the field based on what you're describing. i think this behavior should be explained in our documentation

Reply:
It is explained here https://docs.viam.com/docs/components/motor/#required-attributes---non-encoded-dc-motor